package.json

```diff
@@ -34,7 +34,8 @@
     "windows-release": "^5.1.0",
     "winston": "2.4.4",
     "yargs": "14.2.3",
-    "decompress": "4.2.1"
+    "decompress": "4.2.1",
+    "unzipper": "^0.12.3"
```

Comment:
How heavy is this library?

Reply:
`324KB` is the size of the package. That'll be the addition to `node_modules` for the package.

Reply:
Ack, approving it for now. But let's create a story and migrate from the old `decompress` to the new package or zlib.